pkg/utils/resource_informer.go

```diff
@@ -0,0 +1,126 @@
+package utils
```

Comment:
Seems `sigs.k8s.io/kubefed/pkg/controller/util/resourceinformer.go` is better.

WDYT?

Reply:
Not exactly suitable:
1. the Informer labelSelector is not same.
2. sigs.k8s.io/kubefed/pkg/controller/util/resourceinformer.go UpdateFunc compare not only spec info.
You can check.